### PR TITLE
Modified Makefile to work on ukwiki

### PR DESCRIPTION
**Pull request by HAKSOAT**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,6 +10,11 @@ vocab_limit=
 vocab_str=10k
 
 
+ukwiki_vectors: \
+        datasets/ukwiki-$(dump_date)-plaintext.w_labels.txt \
+        datasets/ukwiki-$(dump_date)-learned_vectors.$(vector_dimensions)_cell.vec.bz2 \
+        datasets/ukwiki-$(dump_date)-learned_vectors.$(vector_dimensions)_cell.$(vocab_str).kv
```
Comment (halfak) [inline Makefile]: You only need line 16 here.  Line 14 and 15 are implied when you have line 16.

Comment (HAKSOAT) [reply Makefile]: Oh. Waoh. Does this mean line 16 will re-run 14 and 15 or just use the data there?

Comment (halfak) [reply Makefile]: It'll just re-use the data there.  Essential what this rule is saying is:

```
In order to satisfy 'ukwiki_vectors', we need to solve all 
dependencies for the plaintext, .vec, and .kv files.
```

When I think you mean to say:

```
In order to satisfy 'ukwiki_vectors', we need to solve all 
dependencies for the .kv file (which implicitly include the 
plaintext and .vec files).
```

**Comment by halfak** [comment]
@HAKSOAT This looks like it is in a good place.  Is it ready for review?

**Comment by HAKSOAT** [comment]
@halfak Yes. However, it is tied to [this task](https://github.com/wikimedia/drafttopic/pull/55/files). So I think when that task is done, then I can be sure of this PR. I was basically only experimenting with cutoff though to see if I'd get better results with the model metrics.

**Comment by halfak** [comment]
OK I'll hang tight then.  Please feel free to ping when you think this is ready to merge :)

**Pull request merged**